src/main/java/org/javamodularity/moduleplugin/tasks/CompileTestTask.java

```diff
@@ -34,6 +34,7 @@ public void configureCompileTestJava() {
         });
     }
 
+    @SuppressWarnings("Convert2Lambda")
```

Comment:
Added this to stop the use of code that can be converted to lambda, but which shouldn't be, showing up as a warning when compiling.